src/NavDropdown.tsx

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import Dropdown from './Dropdown';
+import Dropdown, { DropdownProps } from './Dropdown';
 import NavItem from './NavItem';
 import NavLink from './NavLink';
-import {
-  BsPrefixPropsWithChildren,
-  BsPrefixRefForwardingComponent,
-} from './helpers';
+import { BsPrefixRefForwardingComponent } from './helpers';
+import { PropsFromToggle } from './DropdownToggle';
 
-export interface NavDropdownProps extends BsPrefixPropsWithChildren {
+export interface NavDropdownProps
+  extends DropdownProps,
+    Omit<React.HTMLAttributes<HTMLElement>, 'onSelect' | 'title'>,
+    React.PropsWithChildren<PropsFromToggle> {
```

Comment:
We can drop `React.PropsWithChildren<PropsFromToggle>` actually. I looked thru the code and these props are not needed. Other than that, looks good!